eth/contracts/gnosis-1.3.0/GnosisSafe.sol

```diff
@@ -35,14 +35,22 @@ contract GnosisSafe is
     // keccak256(
     //     "EIP712Domain(uint256 chainId,address verifyingContract)"
     // );
-    bytes32 private constant DOMAIN_SEPARATOR_TYPEHASH = 0x47e79534a245952e8b16893a336b85a3d9ea9fa8c573f3d803afb92a79469218;
+    bytes32 private constant DOMAIN_SEPARATOR_TYPEHASH =
```

Comment:
Wouldn't format anything in `gnosis-1.3.0` - it's a trusted code that we won't be working on